cascading/src/main/java/com/twitter/ambrose/cascading/CascadingJob.java

```diff
@@ -60,69 +64,13 @@ public void setFeatures(String[] features) {
 
   @JsonIgnore
   public void setJobStats(HadoopStepStats stats) {
-    Map<String, Long> counterNameToValue = counterGroupInfoMapHelper(stats);
-
-    // job metrics
-    int totalMappers = stats.getNumMapTasks();
-    int totalReducers = stats.getNumReduceTasks();
-    Map<String, Number> metrics = new HashMap<String, Number>();
-    metrics.put("numberMaps", totalMappers);
```

Comment:
I've opted to reuse the hadoop metric enum names here, instead of create yet another set of strings to identify these values.

I'll be updating client js code to retrieve these metrics using the canonical enum names.